cypress.config.mjs

```diff
@@ -0,0 +1,50 @@
+import { defineConfig } from 'cypress';
+import webpackProcessor from '@cypress/webpack-preprocessor';
+
+import hexToRgb from './test/plugins/hex-to-rgb.js';
+import coveragePlugin from './test/plugins/coverage.js';
+import fakerPlugin from './test/plugins/faker-generator.js';
+import webpackOptions from './test/plugins/webpack.config.js';
+
+fakerPlugin();
```

Comment:
Now that cypress config is js, we can run the run-once stuff in the config vs a plugin-style method.